application/common/sync/Synchronizer.php

```diff
@@ -105,6 +105,12 @@ public function syncAll()
         }
     }
     
+    /**
+     * Synchronize a specific user, requesting their information from the
+     * ID Store and updating it accordingly in the ID Broker.
+     *
+     * @param string $employeeId The EmployeeID of the user to sync.
+     */
     public function syncUser($employeeId)
     {
         $idStoreUser = $this->idStore->getActiveUser($employeeId);
```

Comment:
It may be worth me creating a very simple ValueObject so that it's clear (even to the IDE) how to retrieve the various fields. If it will only take a minimal time commitment, I may do that.